cvat-ui/src/actions/tasks-actions.ts

```diff
@@ -35,6 +35,11 @@ export enum TasksActionTypes {
     UPDATE_TASK_SUCCESS = 'UPDATE_TASK_SUCCESS',
     UPDATE_TASK_FAILED = 'UPDATE_TASK_FAILED',
     HIDE_EMPTY_TASKS = 'HIDE_EMPTY_TASKS',
+    MOVE_TASK_TO_PROJECT = 'MOVE_TASK_TO_PROJECT',
+    MOVE_TASK_TO_PROJECT_SUCCESS = 'MOVE_TASK_TO_PROJECT_SUCCESS',
+    MOVE_TASK_TO_PROJECT_FAILED = 'MOVE_TASK_TO_PROJECT_FAILED',
```

Comment:
UPDATE_TASK, UPDATE_TASK_SUCCESS, UPDATE_TASK_FAILED are not enough?

Reply:
fixed